src/front/glsl/builtins.rs

```diff
@@ -322,7 +323,7 @@ pub fn inject_builtin(
                 let dim_value = image_dims_to_coords_size(dim);
                 let coordinates = make_coords_arg(dim_value + arrayed as usize, Sk::Sint);
 
-                let args = vec![
+                let mut args = vec![
```

Comment:
perhaps we should do `ArrayVec` instead?

Reply:
I would fine with doing this in another PR but I don't think mixing unrelated changes in the same PR would be a good idea.